client/ui/controllers/systemController.cpp

```diff
@@ -92,7 +92,7 @@ QString SystemController::getFileName(const QString &acceptLabel, const QString
 
     mainFileDialog->setProperty("acceptLabel", QVariant::fromValue(acceptLabel));
     mainFileDialog->setProperty("nameFilters", QVariant::fromValue(QStringList(nameFilter)));
-    if (!selectedFile.isEmpty()) {
+    if (QFileInfo::exists(selectedFile)) {
```

Comment:
please return check for an empty string

Reply:
On my computer the file selected is === C:\Users\dvajd\OneDrive\Documents\amnezia.conf (or something like that). I do not have such a file (maybe there is a problem with the Cyrillic alphabet on the way) - it is why I have qml error - qrc:/ui/qml/main2.qml:239:5: QML FileDialog: Cannot set C:/Users/dvajd/OneDrive/?????????/amnezia_for_awg as a selected file because it doesn't exist (error 2). What correct way fix it?

Reply:
This line is not to check if the file exists, because if the file does not exist, it must be created. The error you are talking about is fixed here https://github.com/amnezia-vpn/amnezia-client/pull/636